packages/builder/plugin-swc/src/plugin.ts

```diff
@@ -98,16 +98,19 @@ export const builderPluginSwc = (
       }
 
       const isSSR = utils.target === 'node';
-      const styledComponentsOptions = applyOptionsChain(
-        getDefaultStyledComponentsConfig(isProd, isSSR),
-        config.tools.styledComponents,
-      );
-      swc.extensions.styledComponents = {
-        ...styledComponentsOptions,
-        ...(typeof swc.extensions.styledComponents === 'object'
-          ? swc.extensions?.styledComponents
-          : {}),
-      };
+
+      if (config.tools.styledComponents || swc.extensions?.styledComponents) {
```

Comment:
The `DefaultStyledComponentsConfig` should take effect by default.

Reply:
Pardon, not sure what do you mean

Reply:
Even if the user does not configure `config.tools.styledComponents` or `swc.extensions?.styledComponent`, the `DefaultStyledComponentsConfig` should also be applied.